projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-backup-needs-attention-notice.tsx

```diff
@@ -36,6 +36,13 @@ const useBackupNeedsAttentionNotice = ( redBubbleAlerts: RedBubbleAlerts ) => {
 
 	const noticeTitle = __( 'Oops! We couldn’t back up your site', 'jetpack-my-jetpack' );
 
+	const onCloseClick = useCallback( () => {
+		// Session cookie. Expires at session end.
+		document.cookie = `backup_failure_dismissed=1; SameSite=None; Secure`;
+		delete redBubbleAlerts.backup_failure;
+		resetNotice();
```

Comment:
All of these are very similar in this way, an optimization later on would be to create a utility function called something like `setDismissAndResetNotice` 🤔 Not saying we should do that now, only thinking "out loud"